### PR TITLE
refactor: new folder structure

### PR DESCRIPTION
**Pull request by kbberker**
Hey @jamesdoc! I was thinking of trying to organise the files in some sort of folder structure. Thinking of something like what [Ollie suggests in his video](https://youtu.be/1uQ9j8ynQlM?t=459). 

This is not entirely necessary, so if you would rather keep it the way it is that's fine. Just wanted to get your feedback before continuing.

### CODE REVIEW

**Comment by jamesdoc** [comment]
This looks really good, and will make a nicer happy path to get Jest in 👍

File: src/pages/About/About.jsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import React, { Component } from "react";
 import PropTypes from "prop-types";
-import Icon from "./Icon";
-import "./style/Study.css";
+import Icon from "../../components/Icon/Icon";
+import "../../style/Study.css";
```
Comment (kbberker) [inline src/pages/About/About.jsx:4]: As `Study.css` is used in a number of places I have left it in the top level `style` folder. Maybe the classes could be broken up to the specific components that use them, but don't think that's really necessary.

File: src/pages/Study/Study.jsx [hunk 2]
```diff
@@ -5,14 +5,14 @@ import {
...
-import StudyFooter from "./StudyFooter";
-import StudyLeadImage from "./StudyLeadImage";
-import ExpandableText from "./ExpandableText";
-import Icon from "./Icon";
-import "./style/Study.css";
+import Question from "./components/Question/Question";
+import BiblePopup from "./components/BiblePopup/BiblePopup";
+import StudyHeader from "./components/StudyHeader/StudyHeader";
+import StudyFooter from "./components/StudyFooter/StudyFooter";
+import StudyLeadImage from "./components/StudyLeadImage/StudyLeadImage";
```
Comment (kbberker) [inline src/pages/Study/Study.jsx:12]: The `Study` page had enough of its own components, that I created a subfolder for its components inside the same folder.

File: src/style/App.css [hunk 3]
```diff
@@ -1,106 +1,247 @@
 @import url("https://fonts.googleapis.com/css?family=Roboto:400,500");
-html, body, div, span, applet, object, iframe, h1, h2, h3, h4, h5, h6, hr, p, blockquote, pre, a, abbr, acronym, address, big, cite, code, del, dfn, em, img, ins, kbd, q, s, samp, small, strike, strong, sub, sup, tt, var, b, u, i, center, dl, dt, dd, ol, ul, li, fieldset, form, label, legend, table, caption, tbody, tfoot, thead, tr, th, td, article, aside, canvas, details, embed, figure, figcaption, footer, header, hgroup, menu, nav, output, ruby, section, summary, time, mark, audio, video {
```
Comment (kbberker) [inline src/style/App.css:2]: It looks like Prettier attacked this file in an attempt to make it tidier, but has left it worse.

Would you rather I keep it as before?

File: src/style/SermonList.css [hunk 4]
```diff
@@ -1,30 +1,40 @@
...
+}
+.sermonList__item__star {
+  align-items: center;
+  color: #72a6bf;
+  display: flex;
+  padding: 0 16px;
+}
+.sermonList__item__star:hover {
+  color: #446d81;
+}
```
Comment (kbberker) [inline src/style/SermonList.css:40]: This is all Prettier doing its formatting stuff. Can revert them if you prefer.

**Comment by kbberker** [comment]
@jamesdoc this is now ready for your review! Let me know if there's anything you would like changed. I'll branch off this to start testing.

**Pull request merged**